src/app/static/src/app/store/modelRun/mutations.ts

```diff
@@ -61,6 +60,7 @@ export const mutations: MutationTree<ModelRunState> = {
                 detail: null
             });
         }
+        state.pollingCounter += 1
```

Comment:
Should be an `else` here.

Reply:
There's no need for else if the counter is above.